pkg/controller/service.go

```diff
@@ -160,11 +169,12 @@ func (s *ShadowServiceManager) cleanupPortMapping(oldUserSvc *corev1.Service, ne
 		}
```

Comment:
Shouldn't we do something specific for UDP here? If the port.Protocol isn't UDP, we shouldn't consider it found. no?

Reply:
Good catch!